deploy/helm/kuberhealthy/templates/khcheck-pod-restarts.yaml

```
@@ -59,6 +61,44 @@ spec:
     restartPolicy: Never
     serviceAccountName: pod-restart-sa
 ---
+{{- if .Values.check.podRestarts.allNamespaces }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: pod-restart-rb
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: pod-restart-role
+subjects:
+  - kind: ServiceAccount
+    name: pod-restart-sa
+    namespace: kuberhealthy
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: pod-restart-role
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - events
+    verbs:
+      - list
+{{- end }}
+{{- if not .Values.check.podRestarts.allNamespaces }}
```

wondering if this could simplified to just an `else`
```suggestion
{{ else }}
```